service/src/test/java/bio/terra/pipelines/service/StatusServiceTest.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.pipelines.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.when;
+
+import bio.terra.pipelines.app.configuration.internal.StatusCheckConfiguration;
+import bio.terra.pipelines.dependencies.sam.SamService;
+import bio.terra.pipelines.generated.model.ApiSystemStatusSystems;
+import bio.terra.pipelines.testutils.BaseTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.jdbc.core.ConnectionCallback;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = StatusService.class)
+class StatusServiceTest extends BaseTest {
+  @Autowired private StatusService statusService;
+
+  @MockBean private NamedParameterJdbcTemplate jdbcTemplate;
+  @MockBean private StatusCheckConfiguration configuration;
+  @MockBean private SamService samService;
+
+  @Mock private final JdbcTemplate jdbcTemplateMock = new JdbcTemplate();
+
+  @BeforeEach
+  void setup() {
+    // by default, everything should work. we can override some of these in specific tests
+    when(jdbcTemplate.getJdbcTemplate()).thenReturn(jdbcTemplateMock);
+    when(configuration.enabled()).thenReturn(true);
```

Comment:
i assume you mocked this out instead of pulling it from the test application.yml so you could make this a `BaseTest` or was there another reason?

Reply:
it was easier to change the values via mocking since the configuration doesn't have setters. the speed of using `BaseTest` over `BaseEmbeddedDbTest` is also great but wasn't my motivation originally